src/main/java/uk/ac/ebi/ena/webin/cli/service/RunService.java

```diff
@@ -64,24 +70,33 @@
     getRun( String runId, String userName, String password, boolean test )
     {
         RestTemplate restTemplate = new RestTemplate();
-        restTemplate.setErrorHandler( new NotFoundErrorHandler( WebinCliMessage.RUN_SERVICE_VALIDATION_ERROR.format( runId ),
```

Comment:
I think we should continue to use setErrorHandler and improve the NotFoundErrorHandler. See previous comment.